website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "Operator Observability Best Practices"
+linkTitle: "Observability Best practices"
+weight: 6
+description: This guide describes the best practices concepts for adding Observability to operators.
+---
+
+## Operator Observability Best Practices
+
+In this document we provide best practices and examples for creating metrics, [recording rules](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules) and alerts. It is based on the general guidelines in [Operator Capability Levels](https://sdk.operatorframework.io/docs/overview/operator-capabilities/).
+
+**Note:** For technical documentation of how to add metrics to your operator, please read the [Metrics](https://book.kubebuilder.io/reference/metrics.html) section of the Kubebuilder documentation.
+
+### Operator Observability Recommended Components
+
+1. **Health and Performance metrics** - for all of the operator components  
+    1.1. Metrics should be implemented based on the guidelines below.  
+    1.2. **Metrics Documentation** - All metrics should have documentation.  
+    1.3. **Metrics Tests** - Metrics should include test that verify that they exist and that their value is correct.  
+2. **Alerts** for when things are not working as expected for each of the operator's components  
+    2.1  Alerts should be implemented based on the guidelines below.  
+    2.2. **Alerts Runbooks** - Each alert can include a `runbook_url` annotation and an alert runbook that describes it. See additional details below.  
+    2.3. **Alerts Tests** - Alerts should include E2E Testing and unit tests.  
+3. **Events** - Custom Resources should emit custom events for the operations taking place.
+
+Additional component would be `Dashboards` and `Logs`, which are not addressed in this document at this point.  
```

Comment:
What about adding `tracing` here for future documentation?
(It may not be necessary for all operators, well in terms of "observability", this can be covered.
One use case may be: observing webhooks)

Reply:
Added. Thanks.